components/molecules/TableTile/TableTile.scss

```
@@ -20,7 +20,7 @@
     border-color: $light-gray;
     border-style: solid;
     border-width: 1px 0 0;
-    padding: 1.168rem 1.8rem 1.772rem;
+    padding: 0 1.8rem 0;
```

```suggestion
    padding: 0 1.8rem;
```
This could be simplified a little bit